pandas/io/tests/test_sql.py

```diff
@@ -236,7 +236,7 @@ def _close_conn(self):
         pass
 
 
-class PandasSQLTest(unittest.TestCase):
+class PandasSQLTest(object):
```

Comment:
I would still inherit from our base class for tests (which is not that useful for pytest, but simplifes things)